cmd/updateprs/updateprs.go

```diff
@@ -104,7 +108,7 @@ func runClose(c *cobra.Command, _ []string) {
 	// Prompting for confirmation
 	if !yesFlag {
 		// TODO: add the number of PRs that it will actually close
-		if !p.AskConfirm(fmt.Sprintf("Close all PRs from the %s campaign?", dir.Name)) {
+		if !p.AskConfirm(fmt.Sprintf("Close %s campaign PRs for all repos in %s?", dir.Name, repoFile)) {
```

Comment:
Updating this so that it is in line with the one I added for amend-description. Now that we have `--repos` option we need to specify that it's not _all_ PRs but only the ones in the relevant file.